Gemfile

```diff
@@ -5,4 +5,8 @@ gem 'rb-readline'
 gem 'fastlane'
 gem 'cocoapods'
 
+# Workaround for https://github.com/ffi/ffi/issues/1105
+# https://bundler.io/v2.5/man/gemfile.5.html#FORCE_RUBY_PLATFORM
+gem 'ffi', force_ruby_platform: true
```

Comment:
Hmm seems iOS is simply using an older version... I'm worried about this adding a explicit dependency which we don't actually need... I wonder if we can just downgrade to the same version we have in iOS?

Reply:
Yea that might be simpler. Should be done now!